docs/monitoring/metrics.md

```diff
@@ -801,53 +849,67 @@ Thus, in order to infer the metric identifier:
 <table class="table table-bordered">
   <thead>
     <tr>
-      <th class="text-left" style="width: 20%">Scope</th>
-      <th class="text-left" style="width: 30%">Metrics</th>
-      <th class="text-left" style="width: 50%">Description</th>
+      <th class="text-left" style="width: 18%">Scope</th>
+      <th class="text-left" style="width: 26%">Metrics</th>
+      <th class="text-left" style="width: 48%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <th rowspan="9"><strong>Job (only available on JobManager)</strong></th>
       <td>lastCheckpointDuration</td>
-      <td>The time it took to complete the last checkpoint.</td>
+      <td>The time it took to complete the last checkpoint (in milliseconds).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>lastCheckpointSize</td>
-      <td>The total size of the last checkpoint.</td>
+      <td>The total size of the last checkpoint (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>lastCheckpointExternalPath</td>
       <td>The path where the last external checkpoint was stored.</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>lastCheckpointRestoreTimestamp</td>
-      <td>Timestamp when the last checkpoint was restored at the coordinator.</td>
+      <td>Timestamp when the last checkpoint was restored at the coordinator (in milliseconds).</td>
```

Comment:
"milliseconds" as a unit is not sufficient; it's missing a reference port (like the UNIX epoch).

Reply:
Yes. 
but the "milliseconds" not cause confusion to user

Reply:
Yes.
but the "milliseconds" not cause confusion to the users.